guides/common/docinfo.html

```diff
@@ -1 +1,74 @@
-<!-- docinfo -->
+<!-- Generate a nice TOC -->
+<script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
+<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tocify/1.9.0/javascripts/jquery.tocify.min.js"></script>
```

Comment:
It would be cool if we checkin these scripts. They are ~350kb in total which is a lot, but I'd be ok with that.